packages/psss-server/examples.http

```diff
@@ -39,7 +39,7 @@ content-type: application/json
 DELETE http://localhost:8040/v1/weeklyReport/TO?week=2021W07 HTTP/2.0
 
 ### Fetch Confirmed Weekly Report Data
-GET http://localhost:8040/v1/confirmedWeeklyReport?startWeek=2021W07&endWeek=2021W07 HTTP/2.0
+GET http://localhost:8040/v1/confirmedWeeklyReport?startWeek=2021W07&endWeek=2021W07&orgUnitCodes=TO&orgUnitCodes=FJ HTTP/2.0
```

Comment:
While I get that this works out of the box, I do find it a tad ugly. We have a bit of a convention of providing a comma separated list if we want an array of org units?

Reply:
Just commented on that in parallel - happy to change this!